test/integration/query-interface.test.js

```diff
@@ -586,14 +586,15 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
           expect(table).to.not.have.property('lastName');
         });
       });
-
-      it('should be able to remove a column with a foreign key constraint', function() {
-        return this.queryInterface.removeColumn('users', 'manager').bind(this).then(function() {
-          return this.queryInterface.describeTable('users');
-        }).then(function(table) {
+      if(dialect !== 'mysql'){
```

Comment:
Yes, lets add removeColumn test with schema in general. Other dialects may have same issue.